os/src/main/kotlin/com/gojuno/commander/os/Processes.kt

```diff
@@ -102,13 +105,15 @@ fun process(
         .subscribeOn(io()) // Prevent subscriber thread from unnecessary blocking.
         .observeOn(io())   // Allow to wait for process exit code.
 
-private fun prepareOutputFile(commandAndArgs: List<String>): File = Random()
```

Comment:
Sorry for that unused parameter, idea was to use command name and arguments for name of the file, but I didn't do it because haven't found a good way to 100% escape/encode it properly :(

Reply:
I assume you were kinda confused editing this function :D

Reply:
Not much)